components/layout.tsx

```diff
@@ -40,7 +41,9 @@ export const Layout = ({ children, hideNav, ...meta }: LayoutProps) => (
       <div className="flex flex-col w-screen h-screen text-web">
         {hideNav ? null : <NavBar />}
         <div className="h-screen w-screen">
-          {children}
+          <ReactFlowProvider>
```

Comment:
I've moved the provider wrapper to the top level so we don't have to repeat it everywhere...and since we always have to draw it in the background.